### DISCUSSION TITLE
Embedding Text Font Into .DXF

### DISCUSSION THREAD

**Discussion by geersjake** [haplokuon/netDxf#260] [Q&A] [answered]
Hello,

I am wondering how/if it is possible to embed a particular font into the generated .DXF file. I am using an obscure font for CNC purposes and cannot guarantee consumers of the file would have that font installed. It seems to be supported in the main product by converting the text to geometry but I am unaware of how I could accomplish this with netDxf or if a workaround exists.

Thanks,

**Comment by haplokuon** [comment, marked as answer]
No, fonts cannot be embedded into a DXF file. Also, netDxf cannot convert a text to geometry, you will need other software to do that. In the case of AutoCad the Express Tools have a utility to explode texts for both SHX and TTF fonts. You can also try to import the DXF into another graphic design software such as CorelDraw, Illustrator, ... or you can try the alternative free software such as Inkscape, QCad,... you will need to check them. They may work for TTF fonts not sure about SHX.

**Reply by geersjake** [reply]
Thanks I will look into that.
